templates/base.html

```diff
@@ -10,7 +10,13 @@
     </head>
     <body>
         <div id="header">
-            <div id="headTitle"><a href="/" class="titleLink">MoviePolls</a>{{if .PageTitle}} - {{.PageTitle}}{{end}}</div>
```

Comment:
Why did you remove the PageTitle here?

Reply:
the page title doesn't fit in the new header, will add it back when i know how to add it back into the new header